scripts/jest/preprocessor.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
               }
             : {}
         )
-      ).code;
```

Comment:
removing this ensures you return source maps to Jest, for better errors and debugging. This is why we now have `Foo` in some of the stack trace tests instead of the generic `Component`